db/migration/V020__insert_default_activities.sql

```diff
@@ -0,0 +1,128 @@
+INSERT INTO curriculums (principal_id, title) VALUES
+  (1, 'Professional Mentorship Program');
+
+INSERT INTO activities (title, description_text, curriculum_week, curriculum_day, start_time, end_time, duration, activity_type_id, curriculum_id) VALUES
+  /* week1 -1 */
```

Comment:
SQL comments, rather than being enclosed in /* */ like in JavaScript, are any comments that come after two dashes, like so:

```sql
SELECT * FROM `activities` WHERE `curriculum_id` = 1; -- grab all activities from the first curriculum
```

Also, I've noticed that some of your comments look like `week1 -1` with only one space, and some of your comments look like `week6 - 1` with a space on either side of the hyphen. Can you please reformat the comments to look like:

```sql
  -- Week 1: Day 1
```

That way they all follow a regular structure and use the SQL comment style.

Reply:
Fix it in commit:  https://github.com/OpenTree-Education/rhizone-lms/pull/421/commits/29960118281b09acb11c3e7b2d789412a10c8629